third_party/googletest/webport/build/Makefile

```diff
@@ -0,0 +1,116 @@
+# Copyright 2020 Google Inc. All Rights Reserved.
```

Comment:
An open question is how to name the directories where I put these Emscripten/NaCl specific additions to third-party libraries. Previously I've been using "naclport" in this project. Now, with addition of Emscripten, we need a new name. In this CL I'm using the "webport" word, but I'm open to other suggestions.

Reply:
"webport" is good. The first name popped up in my mind is "platform_port".

Reply:
I like webport.